packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1066,8 +1066,13 @@ contract Validators is
    * @return The group that `account` was a member of at the end of the last epoch.
    */
   function getMembershipInLastEpoch(address account) public view returns (address) {
-    allowOnlyL1();
-    uint256 epochNumber = getEpochNumber();
+    uint256 epochNumber;
+    if (isL2()) {
+      epochNumber = getEpochManager().getCurrentEpochNumber();
```

Comment:
we could encapsulate this if with a helper, in this contract or even in the epochManager (probably not)

Reply:
+1